node/staking.go

```diff
@@ -44,8 +46,12 @@ func (node *Node) UpdateStakingList(stakeInfoReturnValue *structs.StakeInfoRetur
 		}
 		// True if the token is still staked within the locking period.
 		if curEpoch-startEpoch <= lockPeriodCount.Uint64()*lockPeriodInEpochs {
+			blsPubKey := types.BlsPublicKey{}
+			copy(blsPubKey[:32], stakeInfoReturnValue.BlsPubicKeys1[i][:])
```

Comment:
why copy 3 part?

Reply:
because public key is 96 bytes and solidity doesn't support long byte array very well